examples/snippets/modules/ROOT/examples/build-akka-streamlets-java/project/cloudflow-plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.3")
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.4-SNAPSHOT")
```

Comment:
Will this work in CI build? Or should you do an internalRelease and depend on that?

Reply:
CI replaces the version with the cached build. I didn't want to leave this on 1.3.3 b/c it would not work. 
Part of the ongoing dilemma with `master` vs `version` in the docs.  (this is part of the docs snippets)